packages/foreman/rubygem-unicode/rubygem-unicode.spec

```diff
@@ -7,13 +7,13 @@
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.4.4.4
-Release: 4%{?dist}
+Release: 5%{?dist}
 Summary: Unicode normalization library
 Group: Development/Languages
 License: Ruby
 URL: http://www.yoshidam.net/Ruby.html#unicode
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-Source1: https://www.ruby-lang.org/en/about/license.txt
+Source1: license.txt
```

Comment:
the remote-URL-but-not-annexed confused the `srpm`module:

```
TASK [build_package : Build SRPM] *****************************************************************************************************************************************************************************************************
fatal: [rubygem-unicode]: FAILED! => changed=false 
  command: rpmbuild -bs --define _topdir /tmp/tmpiskuazz5 --define _sourcedir /tmp/tmpiskuazz5/SOURCES --define _builddir /tmp/tmpiskuazz5/BUILD --define _srcrpmdir /tmp/tmpiskuazz5 --define _rpmdir /tmp/tmpiskuazz5 --undefine dist /tmp/tmpiskuazz5/rubygem-unicode.spec
  msg: Failed to srpm build
  output: |-
    setting SOURCE_DATE_EPOCH=1615420800
    error: Bad file: /tmp/tmpiskuazz5/SOURCES/license.txt: No such file or directory
  
    RPM build errors:
        Bad file: /tmp/tmpiskuazz5/SOURCES/license.txt: No such file or directory
```

This was the quickest fix, but we might want to fix the module too. :eyes: @ehelms

Reply:
I think this gem isn't even used anymore:
```console
# dnf repoquery --whatrequires rubygem-unicode --quiet
Failed to set locale, defaulting to C.UTF-8
```

Digging further tells me it was dropped from Hammer: https://github.com/theforeman/hammer-cli/pull/376

Reply:
Oh nice, will drop it then

(the bug in `srpm` remains ;-) )

Reply:
I don't think it's a bug: you don't provide the source, or a way to verify the source if it is retrieved. We shouldn't support that.

Reply:
Aye, unicode should be a goner now (https://github.com/theforeman/hammer-cli/commit/327f19163e74fdc1a418c5157c956cd0f6d5fdf2)

Reply:
Well, the file is in git. Setting the `Source0` to an URL just denotes where the file came from.

Reply:
https://github.com/theforeman/foreman-packaging/pull/10145